### PR TITLE
✅ Add visual tests for live story notification label

### PR DESCRIPTION
**Pull request by Enriqe**
Partial for #21714

Adds visual tests for live story notification label, it mocks when a new page is added.

### CODE REVIEW

**Comment by newmuis** [comment]
Can you also add tests for RTL?

File: examples/visual-tests/amp-story/amp-story-live-story.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * limitations under the License.
+ */
+'use strict';
+
+const {verifySelectorsVisible, verifySelectorsInvisible} = require('../../../build-system/tasks/visual-diff/helpers');
+
+module.exports = {
+  'Inserting a new page should show a notification in the last page.': async (page, name) => {
+    const screen = page.touchscreen;
+    await screen.tap(200, 240);
```
Comment (gmajoulet) [inline examples/visual-tests/amp-story/amp-story-live-story.js]: Can we find this line that works on both mobile and desktop so we can save one file of repeated code?

Comment (Enriqe) [reply examples/visual-tests/amp-story/amp-story-live-story.js]: Done.

**Comment by Enriqe** [comment]
PTAL.

**Comment by gmajoulet** [comment]
Side note: having more tests is always good, but I find it a bit extreme to have 12 screenshots and 3 new files for a simple toast message with 0 parameterization. If we were to do that for every single feature, our visual diff tests would get really really hard to maintain :(

Alternative proposal for the next visual diff tests so this message isn't a useless rant:
- One screenshot mobile LTR
- One screenshot desktop RTL
- Testing that the message isn't displayed when the active page isn't the last one could be unit/integration tests
- The tests can add the `dir=rtl` attribute directly from the JavaScript and it propagates to all the components since there is no shadow DOM, no need to add a new file where the only diff is an extra `dir=rtl`

**Comment by newmuis** [comment]
+1 to:

> Testing that the message isn't displayed when the active page isn't the last one could be unit/integration tests

**Comment by Enriqe** [comment]
I gave it another go, let me know what you think!

**Comment by Enriqe** [comment]
Could you guys also approve the Percy diffs? Thanks!

**Pull request merged**